### ISSUE TITLE
Mantle causing display driver crash

### ISSUE THREAD

**Issue by KittehFaye** [SlimeKnights/Mantle#89]
Hello, I ran into this problem when trying to load the Age of Engineering modpack, and after some troubleshooting I was asked to explain my issue here. I will give you as much detail as possible.
computer: HP Elitebook 8470p
Integrated Graphics card: AMD Radeon HD 7570M (with latest driver version)
OS Version: Windows 7 (x64)
Mantle Version: Mantle-1.10.2-0.1.19-universal.jar

latest log: https://pastebin.com/k5ymuKbD

I opened up the output log/console to watch while the game loaded. This line was the last line I saw before my display driver crashed:
[19:11:20] [Client thread/INFO]: Started loading books...

When my display driver crashes, my screen goes black for a moment, and then recovers. However, the minecraft window goes blank/white and no longer responds, even though I can see the game still loading in the console/output window. The game window never recovers, and when I click on it I get the 'this program has stopped responding' error window.

I uninstalled Mantle (and Tinker's Construct because of the dependency) from the modpack, and as soon as I did that I was able to load the modpack no problem. 

If you need anymore information please let me know, I have given as much info as I can think to give.

edit: One more note, no crash report is being generated.

**Comment by rhodesy523** [comment]
I am also having the same problems as KittehFaye, and after discussing on the AoE discord, found the same solution allowed me to launch the pack without Mantle and Tinker's Construct. 

Device: HP Pavilion.
Graphics card: AMD Radeon HD 8400 (With latest drivers)
OS: Windows 8.1 64 bit.
Mantle version: Mantle 1.10.2-1.1.5
Tinker's Construct version: TConstruct 1.10.2-2.6.5

Latest crash report with Mantle and Tinker's Construct added: https://pastebin.com/DYqgDr6P

**Comment by KnightMiner** [comment]
According to your logs, there is a significant amount of stuff done after loading books and it is successful, so that is not the issue. Does it produce an actual crash report in the crash-reports folder? The log is lacking in an actual error before shutting down which does not help.

The last thing in the log is stitching textures, so there is a chance the number of textures in the pack is the issue as Tinkers Construct adds a large amount. Try removing another mod with a lot of textures like Chisel or Immersive Engineering and see if that also resolve the issue.

**Comment by rhodesy523** [comment]
There's no crash report as such, just a log which I will attach to this. I have attempted to launch the modpack again while removing mods to see if it launches without another mod, but at this time only removing the Mantle and Tinker's Construct mods has allowed it to launch. I'll keep you updated.  

Log from launcher without Chisel mod: https://pastebin.com/WqcCsMTt

**Comment by KnightMiner** [comment]
Still hard to say its a Mantle issue. Try playing a pack with just Mantle, or Mantle and Tinkers Construct, if it breaks on its own it could be, but I am still betting this comes down to either too many textures or some mod conflict.

Given that 1.10 Mantle has existed for awhile now with no issue reports and AoE is a new pack, its probably some combination in the pack with your graphics cards (a.k.a. the hardest type of issue to track down)

**Comment by rhodesy523** [comment]
Ok, so after removing Forestry + Forestry extras, Chisel and Immersive Engineering, it launched. It's got me buggered. Might be time to upgrade things anyway. Thanks for the quick response, and helping trouble shoot.

**Comment by KittehFaye** [comment]
After testing out a few different drivers versions I was able to find one that works with the pack, and considering the information rhodesy was able to give, it probably is unfortunately some combination of assets in the pack not playing well with the graphics card, like you said @KnightMiner . Thank you both for the troubleshooting help. Since I found a solution and the problem wasn't with Mantle itself I'll close this issue now :3 
@rhodesy523 If you're still having issues, you might need to find an alternative driver for your card. My husband has the same laptop as mine and was able to open the pack no problem, so he gave me the driver he was using, but he can't remember where he got said driver and looking online for it yields no results, yet it's more up-to-date than the one I was using and works better overall. I originally was getting my drivers from the HP site to match my specific laptop, but apparently windows, hp, and amd all have different driver versions for the same card, so it might just be a case of finding that one that works for you like it was for mine. Good luck!
